cocos/game/splash-screen.ts

```diff
@@ -37,13 +37,15 @@ import { SetIndex } from '../rendering/define';
 import { ccwindow, legacyCC } from '../core/global-exports';
 import { XREye } from '../xr/xr-enums';
 import { PipelineRuntime } from '../rendering/custom';
+import { ResolutionPolicy } from '../ui/view';
 
 const v2_0 = new Vec2();
 type SplashLogoType = 'default' | 'none' | 'custom';
 type SplashBackgroundType = 'default' | 'color' | 'custom';
 type WatermarkLocationType = 'default' | 'topLeft' | 'topRight' | 'topCenter' | 'bottomLeft' | 'bottomCenter' | 'bottomRight';
 
 interface ISplashSetting {
+    policy?: number;
```

Comment:
Why the type is not ResolutionPolicy?

Reply:
ResolutionPolicy is an object, but the value inside the setting is a number